File: Beecrowd/2930 TCC da Depressão Natalino/main.c
#include <stdio.h>

int main(){
    int e, d;

    scanf("%d %d", &e, &d);

    if(e < (d - 2))
        printf("Muito bem! Apresenta antes do Natal!\n");
    else if(e <= d){
        printf("Parece o trabalho do meu filho!\n");
        if((d + 1) < 24)
            printf("TCC Apresentado!\n");
        else
            printf("Fail! Entao eh nataaaaal!\n");
    }
    else
        printf("Eu odeio a professora!\n");        

    return 0;
}

File: Beecrowd/2930 TCC da Depressão Natalino/main.py
e, d = [int(_) for _ in input().split()]

if e < (d - 2):
    print("Muito bem! Apresenta antes do Natal!")
elif e <= d:
    print("Parece o trabalho do meu filho!")
    if (d + 1) < 24:
        print("TCC Apresentado!")
    else:
        print("Fail! Entao eh nataaaaal!")
else:
    print("Eu odeio a professora!")